### PR TITLE
Fixed incorrect positional options setup

### PR DESCRIPTION
**Pull request by janoc**
Simple fix for the incorrect `--config` option parsing in the osvr_server.

Fixes: #571

### CODE REVIEW

File: apps/osvr_server.cpp [hunk 1]
```diff
@@ -61,14 +61,14 @@ int main(int argc, char *argv[]) {
...
-        ("config", "server configuration filename")
+        ("config", opt::value< std::vector<std::string> >(), "server configuration filename")
         ("help", "display this help message")
         ("verbose,v", "enable verbose logging")
         ("debug,d", "enable debug logging");
     optionsAll.add(optionsVisible);
 
     opt::variables_map values;
-    try {
+    try {
```
Comment (rpavlik) [inline apps/osvr_server.cpp]: trailing whitespace - I can fix.

Comment (janoc) [reply apps/osvr_server.cpp]: Sorry about that, Visual Studio is always a PITA with this.

Comment (rpavlik) [reply apps/osvr_server.cpp]: I hear you. Recently edited code on a machine without the clang format extension installed. It was terrible.

**Comment by rpavlik** [comment]
OK, I fixed the whitespace (and while I was at it made the rest clang-formatted, including putting the disable comments around the boost program options setup stuff) Just going to build and test then merge. Thank you!

**Comment by janoc** [comment]
While at it, have a look at #331 too - it is a similar tiny patch and is sitting there since June 2015 ...

**Comment by rpavlik** [comment]
merged! Will see if I can do #331 tomorrow.

**Pull request merged**